library/src/main/res/layout/chucker_fragment_transaction_payload.xml

```diff
@@ -1,31 +1,72 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:animateLayoutChanges="true"
     tools:context="com.chuckerteam.chucker.internal.ui.transaction.TransactionPayloadFragment">
 
-    <ProgressBar
+    <androidx.core.widget.ContentLoadingProgressBar
         android:id="@+id/loadingProgress"
+        style="@style/Widget.AppCompat.ProgressBar"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:indeterminate="true"
-        android:layout_gravity="center"
         android:layout_margin="@dimen/chucker_doub_grid"
-        android:visibility="visible" />
+        android:indeterminate="true"
+        android:visibility="visible"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/emptyPayloadImage"
+        android:layout_width="@dimen/chucker_octa_grid"
+        android:layout_height="@dimen/chucker_octa_grid"
+        android:tint="@color/chucker_color_primary"
+        android:layout_marginBottom="@dimen/chucker_quad_grid"
+        android:contentDescription="@string/chucker_body_empty"
+        android:src="@drawable/chucker_empty_payload"
```

Comment:
Nit: can you please reformat this XML file. The order of the attributes is wrong. It should be:

```suggestion
    <ImageView
        android:id="@+id/emptyPayloadImage"
        android:layout_width="@dimen/chucker_octa_grid"
        android:layout_height="@dimen/chucker_octa_grid"
        android:layout_marginBottom="@dimen/chucker_quad_grid"
        android:contentDescription="@string/chucker_body_empty"
        android:src="@drawable/chucker_empty_payload"
        android:tint="@color/chucker_color_primary"
```

Reply:
Ok, will do. Just skipped that because it also changes spacing.